### ISSUE TITLE
Invalid data provider in the test suite

### ISSUE THREAD

**Issue by Ocramius** [doctrine/common#772]
As per https://github.com/doctrine/common/pull/765#discussion_r99428560 by @Majkl578, the test suite currently loads an invalid data provider:

```
1) Warning
The data provider specified for Doctrine\Tests\Common\Persistence\ObjectManagerDecoratorTest::testAllMethodCallsAreDelegatedToTheWrappedInstance is invalid.
```

See Travis: https://travis-ci.org/doctrine/common/jobs/197349731#L228

**Comment by Majkl578** [comment]
Caused by 6a56df9a2445db166dbdae8079b9608f72a8496e.

Ping @ondrejmirtes.

**Comment by ondrejmirtes** [comment]
Fixed in #773.
